deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,31 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csiDriverOptions defines defines CSI Driver settings applied per cluster.
+  csiDriverOptions:
+    # common settings for all CSI drivers.
+    common:
+      readAffinity:
+        # set enabled to true to enable read affinity.
+        enabled: false
+        # CRUSH location labels define which node labels to use
+        # as CRUSH location. This should correspond to the values set in
+        # the CRUSH map.
+        # Defaults to all the labels mentioned in
+        # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
+        # crushLocationLabels:
+        # - "kubernetes.io/hostname"
+        # - "topology.kubernetes.io/zone"
+        # - "topology.rook.io/rack"
+
+    # cephfs driver specific settings.
+    cephfs:
+      # Set CephFS Kernel mount options to use https://docs.ceph.com/en/latest/man/8/mount.ceph/#options
+      # Set to "ms_mode=secure" when connections.encrypted is enabled in CephCluster CR
+      # kernelMountOptions: "ms_mode=secure"
+      # Set CephFS Fuse mount options to https://docs.ceph.com/en/quincy/man/8/ceph-fuse/#options
+      # fuseMountOptions: ""
```

Comment:
> Is there any special behavior users or Rook need to handle here? Does setting `fuseMountOptions` automatically select the FUSE driver? It might be good to note that for users regardless of the answer.
> 
> Either, "setting fuseMountOptions automatically selects the FUSE driver," or "fuseMountOptions only applies when the FUSE driver is enabled"
> 
> I also presume that since I don't see kernel/fuse mount options here, that means that kernel/fuse enable/disable is a common setting for all ceph clusters?

These settings only pass the mountoptions globally for the cephfs driver in a single ceph cluster. 
Enabling disabling/ auto selection depends on the storage class(cluster specific).